web/src/pages/Profile/Votes/VoteCard/CaseStatus.tsx

```
@@ -0,0 +1,62 @@
+    case Periods.vote:
+      return "Voting";
+    case Periods.appeal:
+      return "Crowdfunding Appeal";
+    case Periods.execution:
+      return "Closed";
+    default:
+      return "In Progress";
+  }
+};
+
+const CaseStatus: React.FC<ICaseStatus> = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)